### DISCUSSION TITLE
(Paid commission) Arcane World Ritual Table doesn't consume items properly depending on their slot location.

### DISCUSSION THREAD

**Discussion by Ski-z** [ACGaming/UniversalTweaks#173] [Bugfix Suggestions]
In Arcane World, there's an awesome Crafttweaker-scriptable GUI block called the Ritual Table.
I've worked hard to incorporate this table into my modpack, only to discover a major bug.

The table is "Powered" by items I define in my Crafttweaker script. These items should be consumed when the "ritual" is cast (button is pressed). This works just fine in most cases, but I found that when I place items in the right-most slots of the GUI (recpies are shapeless), the items are not consumed. Here's a video: https://streamable.com/27yv5d

Here's a test script you can use:

```
import mods.ArcaneWorld;
import crafttweaker.block.IBlock;
import crafttweaker.block.IBlockDefinition;
import crafttweaker.item.IItemDefinition;
import crafttweaker.block.IBlockState;
import crafttweaker.entity.IEntity;
import crafttweaker.event.IBlockEvent
import crafttweaker.item.IItemStack;
import crafttweaker.player.IPlayer;
import crafttweaker.world.IBlockPos;
import crafttweaker.world.IWorld;

ArcaneWorld.createRitualCommand("potion_instahealth", "Effect of Healing", ["/effect @a[r=6] minecraft:instant_health 20","/effect @a[r=6] minecraft:regeneration 300", "/particle witchMagic ~.5 ~1 ~.5 0.2 1 0.2 0 1000 force @a","/playsound minecraft:entity.player.levelup master @a ~ ~ ~ 1.5 0.75 0.5"], <minecraft:gold_ingot>,<minecraft:quartz>);
ArcaneWorld.createRitualTime("Ritual_Time", "Ritual of Time", 12000, <minecraft:stick>,<minecraft:quartz>);
```

____

Mods you'll need to test: 

Arcane World: https://www.curseforge.com/minecraft/mc-mods/arcane-world/download/2972860
LemonLibrary: https://legacy.curseforge.com/minecraft/mc-mods/lemonlib/files/2639879
CraftTweaker: https://www.curseforge.com/minecraft/mc-mods/crafttweaker/download/4459202
HEI: https://www.curseforge.com/minecraft/mc-mods/had-enough-items/download/4506796

**Comment by ACGaming** [comment]
Referred to [RotN fork](https://legacy.curseforge.com/minecraft/mc-mods/arcane-world-rotn-edition).
